src/NServiceBus.AcceptanceTests/Routing/When_broadcasting_a_command.cs

```diff
@@ -109,10 +111,12 @@ public class MyMessageHandler : IHandleMessages<Request>
             {
                 public Task Handle(Request message, IMessageHandlerContext context)
                 {
+                    var options = new ReplyOptions();
+                    options.RouteReplyToThisInstance();
```

Comment:
This is more precise than *self* which can mean either this *instance* or this *endpoint*.